scripts/imgtool/keys/__init__.py

```diff
@@ -0,0 +1,62 @@
+"""
```

Comment:
What is this with a file called `__init__.py`, I haven't seen that before.

Reply:
Every package needs one and it works similarly to `__init__` in classes. When a package is imported `__init__.py` is run.

Reply:
(Though note the requirement for `__init__.py` was dropped in Python 3)